lib/Parser/Constants.php

```diff
@@ -82,6 +83,7 @@ abstract class Constants
         "AMD"       => [ "radv" ],
         "Qualcomm"  => [ "tu" ],
         "Broadcom"  => [ "v3dv" ],
+        "Google"  => [ "vn" ],
```

Comment:
This should be Emulation. For virgl, that has the same purpose but for OpenGL the vendor is not the developer because vendor in fact mentions the hardware vendor, that could be or not at the same time the developer.

Reply:
ok, done